hvpy/api_groups/jpeg2000/get_jp2_image.py

```diff
@@ -0,0 +1,51 @@
+from typing import Optional
+from datetime import datetime
+
+from pydantic import validator
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+class getJP2ImageInputParameters(HvpyParameters):
+    """
+    Handles the input parameters of the `getJP2Image API.
+
+    <https://api.helioviewer.org/docs/v2/api/api_groups/jpeg2000.html#getjp2image>`__.
```

Comment:
Right, the pre-commit check does that, this will not work properly for the documentation build.

We will need to decide what to do about this later on.